Makefile

```diff
@@ -187,29 +196,27 @@ endif
 $(eval TMP_ODIR := $(shell mktemp -d))
 gen-manifests: manifests kustomize ## Generates manifests for deploying the operator into $(TARGET)-flotta-operator.yaml
 	$(Q)cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
-ifeq ($(TARGET), k8s)
-	$(KUSTOMIZE) build config/k8s > $(TMP_ODIR)/$(TARGET)-flotta-operator.yaml
-else ifeq ($(TARGET), ocp)
-	$(KUSTOMIZE) build config/ocp > $(TMP_ODIR)/$(TARGET)-flotta-operator.yaml
-else ifeq ($(TARGET), kind)
-	$(KUSTOMIZE) build config/kind > $(TMP_ODIR)/$(TARGET)-flotta-operator.yaml
+ifneq (,$(filter $(TARGET), k8s all))
+	$(Q)$(KUSTOMIZE) build config/k8s > $(TMP_ODIR)/k8s-flotta-operator.yaml
+	$(Q)echo -e "\033[92mDeployment file: $(TMP_ODIR)/k8s-flotta-operator.yaml\033[0m"
+endif
+ifneq (,$(filter $(TARGET), ocp all))
+	$(Q)$(KUSTOMIZE) build config/ocp > $(TMP_ODIR)/ocp-flotta-operator.yaml
+	$(Q)echo -e "\033[92mDeployment file: $(TMP_ODIR)/ocp-flotta-operator.yaml\033[0m"
+endif
+ifeq ($(TARGET), kind)
```

Comment:
Kind target is deleted, no?

Reply:
it was still referenced in several places. updated its removal in a new commit.